client/a8c-for-agencies/sections/referrals/primary/referrals-overview/index.tsx

```diff
@@ -92,6 +98,26 @@ export default function ReferralsOverview( {
 							onClose={ () => setReferralEmail( '' ) }
 						/>
 					) }
+					{ actionRequiredNotice && (
+						<div className="referrals-overview__notice">
+							<NoticeBanner
```

Comment:
I haven't tested this since my account is closed, but the notice should never be shown once the user dismisses it. Is that the case now?

Reply:
No, I actually thought that we want to show this notice all the time, when there's a problem with the payment. This way he would be always aware why he can't make referrals (meaning, Referrals page isn't useful without payment method fully set up)

For example, user dismissed the banner, set up account, and after some time there's some problem on tipalti he needs to address. I think banner would be helpful rather than just disabled button. What do you think?

Reply:
@madebynoam What do you think? Should we show this banner always when there's an error or should it be dismissible?

Reply:
I think for the disabled button, we can show the tooltip/popover we implemented here: https://github.com/Automattic/wp-calypso/pull/92546

What do you think?

Reply:
Sure, let me do that and will see how it feels.

Reply:
@andrii-lysenko and @yashwin all good to leave it dismissible for the current visit:

Here are some examples:
* They land here they see it and dismiss it
* But when they reload or revisit the page it shows again until the issue is resolved

What do you think?